src/core/particle_filter.h

```diff
@@ -1,3 +1,9 @@
+/**
```

Comment:
Global: This file is not related to the tinySLAM method. It was included to the public sources by mistake.